config/routes/api.yaml

```diff
@@ -2,3 +2,9 @@ api_example:
   resource: 'api/example.yaml'
   prefix: /api
   name_prefix: api_example_
+
+api_space:
+  resource: 'api/space.yaml'
+  prefix: /api
```

Comment:
acho que o prefixo aqui já poderia carregar o `/api/spaces` e no arquivo `space.yaml` não precisaria dessa palavra no `path`

Reply:
Concordo com o primeiro ponto, no entanto é necessário ter o _path_, como pode ver na imagem abaixo.
![image](https://github.com/user-attachments/assets/9cc5047c-e3d2-4e17-b046-0f99cc6be2d2)

Reply:
Usei isso aqui também https://symfony.com/doc/current/routing.html#route-groups-and-prefixes